src/views/mfa-verify/CallAndSMSForm.js

```diff
@@ -57,19 +57,19 @@ define(['okta', 'vendor/lib/q', 'views/shared/TextBox'], function (Okta, Q, Text
         className: 'button ' + buttonClassName,
         title: formSubmit,
         click: function () {
-          self.clearErrors();
+          form.clearErrors();
+          this.disable();
+          this.options.title = formSubmitted;
+          this.render();
           // To send an OTP to the device, make the same request but use
           // an empty passCode
           this.model.set('answer', '');
           this.model.save()
-          .then(_.bind(function () {
-            this.options.title = formSubmitted;
```

Comment:
In case of error, we also run this function and change the title.

However, if a connection is bad, the button will not be disabled instantly and user is able to click it multiple times.

![screen shot 2017-05-18 at 5 26 29 pm](https://cloud.githubusercontent.com/assets/17706432/26207070/306cf7e8-3bef-11e7-8411-48ae427d9b86.png)